contingency/contingency-api/src/main/java/com/powsybl/contingency/BranchContingency.java

```diff
@@ -31,7 +31,7 @@ public ContingencyElementType getType() {
     }
 
     @Override
-    public AbstractTrippingTask toTask() {
+    public AbstractTripping toTask() {
```

Comment:
toModification() ? 
No added by this PR, but it is often considered as a bad practice to expose abstract classes in APIs